package.json

```diff
@@ -72,7 +72,8 @@
     },
     "modulePathIgnorePatterns": [
       "<rootDir>/dist/"
-    ]
+    ],
+    "testEnvironment": "jsdom"
   },
   "resolutions": {
```

Comment:
💡 You can also force resolutions to specific versions here. It tends to be a little less risky compared up updating major versions.

Reply:
Thanks, @jpellizzari I _think_ I have it figured out.